var/spack/repos/builtin/packages/py-dotmap/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyDotmap(PythonPackage):
+    """`DotMap` is a dot-access `dict` subclass that
+
+    -   has dynamic hierarchy creation (autovivification)
+    -   can be initialized with keys
+    -   easily initializes from `dict`
+    -   easily converts to `dict`
+    -   is ordered by insertion
+
+    The key feature is exactly what you want: dot-access"""
```

Comment:
Can you run `spack info py-dotmap` and make sure this formats okay? Last I knew, we removed newline chars, so this will all get wrapped onto a single line and the bullet points won't be legible.

Reply:
True, thanks! Adjusted that